### PR TITLE
Updating changelog for hipblas 0.54.0

### PR DESCRIPTION
**Pull request by daineAMD**
Commits: https://github.com/rocmsoftwareplatform/hipblas/compare/release/rocm-rel-5.4...develop

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,10 +3,24 @@
 
 ## (Unreleased) hipBLAS 0.54.0
 ### Added
+- added option to opt-in to use __half type for fp16 for c++ users who define HIPBLAS_USE_HIP_HALF
+- added scritps to plot performance for multiple functions
```
Comment (TorreZuk) [inline CHANGELOG.md:7]: scripts

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,10 +3,24 @@
 
 ## (Unreleased) hipBLAS 0.54.0
 ### Added
+- added option to opt-in to use __half type for fp16 for c++ users who define HIPBLAS_USE_HIP_HALF
```
Comment (TorreZuk) [inline CHANGELOG.md:6]: Maybe make more explicit, this is "to use __half for hiblasHalf in the API" or some wording to indicate this is API change.

**Review by TorreZuk — APPROVED** [review]
Minor points

File: CHANGELOG.md (hunk 2 above)
Comment (daineAMD) [reply CHANGELOG.md]: Changed this, let me know if you have any other concerns.

**Pull request merged**